password-update-service-samples/hibp-integration-ballerina/main.bal

```diff
@@ -0,0 +1,109 @@
+import ballerina/http;
+import ballerina/crypto;
+import ballerina/io;
+
+listener http:Listener proxyListener = new (3000);
+
+service / on proxyListener {
+
+    resource function get .() returns json {
+        return {
+            "message":"Pre-password update service up and running!",
```

Comment:
refer the comment above

Reply:
Fixed with 0a6405027d81ee3b63b9bc16b80ededc1973d230